DeeployTest/testUtils/core/execution.py

```
@@ -191,15 +197,31 @@ def run_simulation(config: DeeployTestConfig, skip: bool = False) -> TestResult:
 
     log.debug(f"[Execution] Simulation command: {' '.join(cmd)}")
 
-    result = subprocess.run(cmd, capture_output = True, text = True, env = env)
-
-    if result.stdout:
-        print(result.stdout, end = '')
-    if result.stderr:
-        print(result.stderr, end = '', file = sys.stderr)
+    cmd_str = " ".join(cmd)
+    process = subprocess.Popen(cmd_str,
```

Ruff (0.15.6)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)